api-client/src/utils/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { Parser, ParserOptions } from 'htmlparser2'
+
+export const parseHTML = (html: string, options: ParserOptions = { decodeEntities: true }): string => {
```

Comment:
Should we rename this to `decodeHtmlEntities`? If you look at Tünews model it looks a little bit weird. We are just passing a string to parseHtml and get a string back.

Reply:
I do not agree here. We use this to parse the content of html which includes strings with no html elementes apart from html entities. Nevertheless, it is also used in some places to parse actual html content, so I'd vote to keep the `parseHTML` naming. However, since you already changed it I won't really insist.

Reply:
It is actually used to parse just strings from html foe the tünews.

Reply:
renamed to `parseHTMLEntities`